### PR TITLE
Don't highlight stdio constants when followed by =

### LINKED ISSUE #155: Don't highlight std(in|out|err) on the LHS of =
It would be nice if in the following example,
screen shot 2018-09-08 at 11 30 34 am
`stdout` was not highlighted, since it isn't being used as a value.

### PR DESCRIPTION
**Pull request by ararslan**
Fixes #155.

After:
screen shot 2018-09-08 at 9 56 17 pm

EDIT: I just realize it isn't `run` that has the keyword arguments, it's `pipeline`, which often occurs within `run`. But whatevs ¯\\\_(ツ)\_/¯

### CODE REVIEW

File: syntax/julia.vim [hunk 1]
```diff
@@ -238,7 +238,7 @@ syntax match   juliaConstBool		display "\<\%(true\|false\)\>"
 syntax match   juliaConstEnv		display "\<\%(ARGS\|ENV\|OS_NAME\|ENDIAN_BOM\|LOAD_PATH\|VERSION\|JULIA_HOME\|PROGRAM_FILE\)\>"
 syntax match   juliaConstEnv1011	display "\<DEPOT_PATH\>"
 syntax match   juliaConstIO06		display "\<\%(STD\%(OUT\|IN\|ERR\)\|DevNull\)\>"
-syntax match   juliaConstIO1011		display "\<\%(std\%(out\|in\|err\)\|devnull\)\>"
+syntax match   juliaConstIO1011		display "\<\%(std\%(out\|in\|err\)\|devnull\)\>\%(\s*=\)\@!"
```
Comment (carlobaldassi) [inline syntax/julia.vim]: This will also remove highlighting for `==` and `===`, e.g. `if stdout == stderr`.

**Comment by carlobaldassi** [comment]
In principle, we should do this for every constant, right? Which doesn't seem too appealing. Also, we don't capture other assignment situations, like tuple unpacking `(stdout, stderr) = (a, b)` or using a constant name for a function, e.g. `function stdout() end`.

It seems like this is justified just to special-case `run`/`pipeline`'s keyword arguments?

File: syntax/julia.vim (hunk 1 above)
Comment (ararslan) [reply syntax/julia.vim]: Oh, you're right 😞

**Comment by ararslan** [comment]
That's a very fair point. I'm not sure there's a sufficiently general solution to this. Special-casing the stdio-related keyword arguments to `pipeline` seems okay I guess, but then it ends up behaving inconsistently with e.g. a user-defined function that has `stdout` as a keyword argument.

**Comment by ararslan** [comment]
Yeah, I think we should just leave this as-is.

**Pull request closed without merging**